lib/BlockedBy.js

```diff
@@ -154,6 +167,13 @@ class BlockedBy extends NodeResque.Plugin {
         // enqueueIn uses milliseconds
         await this.queueObject.enqueueIn(
             this.reenqueueWaitTime() * 1000 * 60, this.queue, this.func, this.args);
+
+        await helper.updateBuildStatus({
+            redisInstance: this.queueObject.connection.redis,
+            buildId,
+            status: 'BLOCKED',
+            statusMessage: `Blocked by these running build(s): ${blockingBuildIds}`
```

Comment:
so user will literally see `Blocked by....: 122, 123` ?

I would want this to a links so that user can lick on them and be taken to the blocking build. Perhaps UI can parse this message and render.

Reply:
Right, I imagine the UI can easily set it to a link?